stock_quant_manual_assign/__manifest__.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2015 Mikel Arregi - AvanzOSC
+# Copyright 2017 ForgeFlow - Jordi Ballester
+# Copyright 2018 Fanha Giang
+# Copyright 2018 Tecnativa - Vicent Cubells
+# Copyright 2016-2018 Tecnativa - Pedro M. Baeza
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+{
+    "name": "Stock - Manual Quant Assignment",
+    "version": "14.0.1.0.0",
+    "category": "Warehouse",
+    "license": "AGPL-3",
+    "author": "AvanzOSC, "
+    "Tecnativa, "
+    "ForgeFlow, "
+    "Fanha Giang, "
+    "Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/stock-logistics-warehouse",
+    "depends": ["stock", "sale"],
```

Comment:
Don't add `sale`. The fail you mention is because you are testing once installed. The test here in Travis is done while installing.

Reply:
because of "sale_line_warn" in product.template is "required=True"
in the **test_stock_quant_manual_assign.py**, it is fail
```
        self.product = self.env["product.product"].create(
            {"name": "Product 4 test", "type": "product"}
        )
```

Reply:
Please read my previous comment. You are testing the bad way, with the module already installed.

Reply:
Thanks, I got it. removed the save. :)